packages/scandipwa/src/component/ProductGallery/ProductGallery.style.scss

```diff
@@ -16,6 +16,9 @@
 }
 
 .ProductGallery {
+    --carousel-scroll-item-height: 38px;
+    --carousel-scroll-margin-top: 28px;
```

Comment:
Move the variable declaration to the `:root` section.
Please see how it's already done in other components.